lib/jekyll/netlify/environment.rb

```diff
@@ -0,0 +1,22 @@
+module Jekyll
+  module Netlify
+    class Environment
+      attr_reader :jekyll_env
+      def initialize
+        @netlify_context = ENV["CONTEXT"]
+        if ENV["DEPLOY_URL"] && (!ENV['JEKYLL_ENV'] || ENV['JEKYLL_ENV'].empty?)
```

Comment:
`ENV["DEPLOY_URL"]` -> `ENV['DEPLOY_URL']`

but ... also .. this is confusing.

Create a new bool `is_netlify?`